### ISSUE TITLE
websocket randomly disconnects while watching dashboard

### ISSUE THREAD

**Issue by wiz** [mempool/mempool#1409]
screen recording of the bug reproducing with JS console:
https://bitcointv.com/w/nGGqvqpb5jeR8XBZSzsvan

**Comment by softsimon** [comment]
Can confirm. 

Connect VPN, disconnect VPN and it will get stuck in an endless force disconnect/connect.

<img width="1029" alt="Screen Shot 2022-03-19 at 23 17 22" src="https://user-images.githubusercontent.com/8561090/159135401-e6c62939-30c3-4e9b-b7a3-46db68f2b88e.png">

**Comment by softsimon** [comment]
> is this fixed now?

I don't think so. It looks like a state that the app can arrive into .. It's just difficult to reproduce as I have not been able to.
